src/ngraph/runtime/cpu/pass/cpu_rnn_fusion.cpp

```diff
@@ -242,10 +242,12 @@ void ngraph::runtime::cpu::pass::LSTMFusion::construct_lstm_fprop()
         auto ht_output = std::make_shared<op::GetOutputElement>(lstm, 0);
         auto ct_output = std::make_shared<op::GetOutputElement>(lstm, 1);
 
+        if (lstm->get_outputs().at(0).get_inputs().size() != 2)
+        {
+            throw ngraph_error("Lstm node doesnt have two outputs");
```

Comment:
The error message was a bit ambiguous to me at first look, maybe something like:
`throw ngraph_error("Lstm node requires two outputs");`?